mirdata/salami.py

```diff
@@ -89,30 +89,42 @@ def _load_metadata(data_home):
 
 
 class Track(object):
-    """SALAMI Track class
+    """salami Track class
 
     Args:
         track_id (str): track id of the track
-        data_home (str): Local path where the dataset is stored.
+        data_home (str): Local path where the dataset is stored. default=None
             If `None`, looks for the data in the default directory, `~/mir_datasets`
 
     Attributes:
+        annotator_1_id (str): TODO
+        annotator_1_time (str): TODO
+        annotator_2_id (str): TODO
+        annotator_2_time (str): TODO
+        artist (str): TODO
+        audio_path (str): TODO
+        broad_genre (str): TODO
+        duration (float): TODO
+        genre (str): TODO
+        sections_annotator1_lowercase_path (str): TODO
+        sections_annotator1_uppercase_path (str): TODO
+        sections_annotator2_lowercase_path (str): TODO
+        sections_annotator2_uppercase_path (str): TODO
+        source (str): TODO
+        title (str): TODO
         track_id (str): track id
-        audio_path (str): audio path of the track
-        source
-        annotator_1_id
-        annotator_2_id
-        duration: duration of the track
-        title: title of the track
-        artist
-        annotator_1_time
-        annotator_2-time
-        broad_genre
-        genre:
-        sections_annotator_1_uppercase: annotation
-        sections_annotator_1_lowercase: annotation
-        sections_annotator_2_uppercase: annotation
-        sections_annotator_2_lowercase: annotation
+
+    Cached Properties:
+        sections_annotator_1_lowercase (SectionData): TODO
+        sections_annotator_1_uppercase (SectionData): TODO
+        sections_annotator_2_lowercase (SectionData): TODO
+        sections_annotator_2_uppercase (SectionData): TODO
+
+    Properties:
+        audio: TODO
```

Comment:
Cached Properties:
        sections_annotator_1_lowercase (SectionData): annotations in hierarchy level 1 from annotator 1
        sections_annotator_1_uppercase (SectionData): annotations in hierarchy level 0 from annotator 1
        sections_annotator_2_lowercase (SectionData): annotations in hierarchy level 1 from annotator 2
        sections_annotator_2_uppercase (SectionData): annotations in hierarchy level 0 from annotator 2
    Properties:
        audio: audio file, sample rate